source/extensions/quic/server_preferred_address/server_preferred_address.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include "source/common/quic/envoy_quic_server_preferred_address_config_factory.h"
+
+namespace Envoy {
+namespace Quic {
+
+class ServerPreferredAddressConfig : public Quic::EnvoyQuicServerPreferredAddressConfig {
+public:
+  struct FamilyAddresses {
```

Comment:
I know this struct is simply being moved so this predates you... But "FamilyAddress" is a strange name for a struct which doesn't really have anything to do with families. Maybe "PreferredAddress"? MappedAddress? Not sure, but "FamilyAddress" seems odd. WDYT?

Reply:
It's for an IP address family (v4 or v6). Maybe `IpVersionConfig` (since it's already in a class named `ServerPreferredAddressConfig`)?